docs/src/main/asciidoc/rest-json-guide.adoc

```diff
@@ -484,6 +484,22 @@ If you want to use the `keycloak` Quarkus extension in your project, you should
 configuration. See link:keycloak-guide#configuring-cors[the Keycloak guide] for more details.
 ====
 
+== GZip Support
+
+Quarkus comes with GZip support (even though it is not enabled by default). The following configuration knobs allow to configure GZip support.
+
+[source, properties]
+----
+quarkus.resteasy.gzip.enabled=true // <1>
+quarkus.resteasy.gzip.max-input=10M // <2>
+----
+
+<1> Enable Gzip support.
+<2> Configure the upper limit on deflated request body. This is useful in limiting risks of attack related by packaging
+the attack in an entity that expands greatly. The default value is `10M`.
```

Comment:
I would try something like "This is useful to mitigate potential attacks by limiting their reach."

Reply:
Thanks. Suggestion taken into account.